l10n_jp_hr_employee/models/qualification_name.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class QualificationName(models.Model):
+    _name = "qualification.name"
```

Comment:
I find it a bit strange as a model name. How about `hr.qualification` or just `qualification`?

Reply:
`name` field is exist in  `hr.qualification` model and this model will have different type of qualifications. So I decided to use `hr.qualification.type`.